tests/tests_unit/test_api/test_relationships.py

```diff
@@ -176,42 +189,63 @@ def test_delete_multiple(self, mock_rel_response):
         assert res is None
 
     def test_advanced_list(self, mock_rel_response):
-        res = REL_API.list(source_resource="asset", relationship_type="belongs_to")
+        res = REL_API.list(source_types="asset", labels=LabelFilter(contains_any=["label_ext_id"]))
         assert {
-            "filter": {"sources": [{"resource": "asset"}], "relationshipTypes": ["belongs_to"]},
-            "limit": 25,
+            "filter": {"sourceTypes": "asset", "labels": {"containsAny": [{"externalId": "label_ext_id"}]}},
+            "limit": 100,
             "cursor": None,
         } == jsgz_load(mock_rel_response.calls[0].request.body)
         assert mock_rel_response.calls[0].response.json()["items"] == res.dump(camel_case=True)
 
     def test_source_target_packing(self, mock_rel_response):
         res = REL_API.list(
-            source_resource="asset",
-            source_resource_id="bla",
-            target_resource="timeseries",
-            target_resource_id="foo",
-            relationship_type="belongs_to",
+            source_types="asset",
```

Comment:
you specify this as List[str] but then call it with str  - it's nice to accept both, but make sure it can in terms of api, and update docs

Reply:
Great catch!